pages/documentation/migration-V3.md

```diff
@@ -277,12 +275,12 @@ const uswds = require("@uswds/compile");
     - **Packages:** `@uswds/uswds/packages`
 1. Recompile your Sass as usual and check that your files compiled as expected. When it compiles successfully, it's using USWDS 3.0!
 </div>
-<!-- End custom Gulp workflow section --> 
+<!-- End custom gulp workflow section --> 
 
-<!-- Start custom Gulp workflow section -->
+<!-- Start custom gulp workflow section -->
 <h4 class="usa-accordion__heading">
   <button class="usa-accordion__button" aria-controls="m-a9">
-    If you're using Webpack
```

Comment:
`webpack` is not capitalized on the webpack website https://webpack.js.org/

Reply:
agreed